COMPILING_ON_LINUX.txt

```diff
@@ -10,7 +10,7 @@ HOW TO BUILD ON UBUNTU:
  $ sudo apt install git build-essential libsdl2-2.0-0 libsdl2-dev libjansson-dev libexpat1-dev libcurl4-openssl-dev libpng-dev libjpeg-dev libspeex-dev libspeexdsp-dev
 
 - For Fedora 25+:
- $ sudo dnf install mesa-libGL-devel SDL2-devel make gcc jansson-devel expat-devel libcurl-devel libpng-devel libjpeg-turbo-devel speex-devel speexdsp-devel
+ $ sudo dnf install vim-common pcre-devel mesa-libGL-devel SDL2-devel make gcc jansson-devel expat-devel libcurl-devel libpng-devel libjpeg-turbo-devel speex-devel speexdsp-devel
```

Comment:
Why is vim-common required ? It sounds a bit weird.

Reply:
It provides 'xxd' which we currently use for json -> C arrays at compile time.

Reply:
Cool, didn't know that.